### ISSUE TITLE
docker-machine 0.6.0 incompability

### ISSUE THREAD

**Issue by xbtsw** [Parallels/docker-machine-parallels#43]
When used with docker-machine 0.6.0, will get a "unexpected EOF" error from driver.

```
Creating machine...
(dev) Calling .Create
(dev) No default boot2docker iso found locally, downloading the latest release...
Error creating machine: Error in driver during machine creation: unexpected EOF
open : no such file or directory
notifying bugsnag: [Error creating machine: Error in driver during machine creation: unexpected EOF]
```

**Comment by legal90** [comment]
Looks similar to #42 
I couldn't catch it yet, but seems like we have to rebuild the plugin with libmachine v0.6.0.

@xbtsw How have you installed our plugin: via `brew` or by downloading a binary from github? Also please, try to reproduce the issue with both of these ways - does the error exist?
https://github.com/Parallels/docker-machine-parallels#installation

**Comment by xbtsw** [comment]
Yes indeed I used the homebrew one. I will try to build from source and report back.

**Comment by legal90** [comment]
> I will try to build from source and report back.

Thanks! Please, try this one too:
https://github.com/Parallels/docker-machine-parallels/releases/download/v1.1.1/docker-machine-driver-parallels

**Comment by kad** [comment]
Out of curiosity, @xbtsw  which versions of other docker components you have installed from homebrew ?

I have 
```
$ brew list --versions | grep docker
docker 1.10.2
docker-machine 0.6.0
docker-machine-parallels 1.1.1_1
```
and I'm able to create new virtual machines without errors.

**Comment by legal90** [comment]
@xbtsw Do you have any updates on this? Does our pre-compiled binary work fine for you?
https://github.com/Parallels/docker-machine-parallels/releases/download/v1.1.1/docker-machine-driver-parallels

**Comment by legal90** [comment]
It should be fixed in [v1.2.0](https://github.com/Parallels/docker-machine-parallels/releases/tag/v1.2.0)
